### DISCUSSION TITLE
Deskflow is the new name for 'Synergy Community Edition'

### DISCUSSION THREAD

**Discussion by nbolton** [deskflow/deskflow#7517] [Announcements]
We have decided to rename the upstream community project which Synergy is built from. 

# Deskflow
<img src="https://github.com/user-attachments/assets/e39e7c75-923d-4620-be6c-6e26e2ed2fb0" width="50">

**Deskflow** is the new name for Synergy Community Edition.

### Why the project rename?

We wanted to draw a clear line between the community and commercial interests; it is important to have somewhere the community can have freedom, without the involvement of a business and the constraints and conflict that come with it.

Deskflow is community-driven with separate interests and motivations from downstream Synergy. Deskflow is still sponsored by Synergy, but the majority of the leadership will be non-Synergy.

### Our Values for Deskflow

- Motivated by the community interests (not business-driven)
- Privacy by default (no update check or phone home on by default)
- Nothing customer-related (this is all moved downstream to Synergy)
- Have fun; we don't need to worry about impressing anyone

# Checklist (priority order):
- [x] Create new logo: [Brainstorming](https://lucid.app/lucidspark/5d2b8666-c5fd-43db-bc03-6b8e4e4af2cd/edit?beaconFlowId=F0084A30AAC99F43&invitationId=inv_a057afe0-a23f-4b9b-aa47-e0b7749fa456&page=0_0#)
- [x] Rename 'Synergy' to 'Deskflow': https://github.com/deskflow/deskflow/pull/7519
- [x] Fix CI (new SonarCloud account, etc)
- [x] Update logos and icons
- [x] Ask permission before update check (default off)
- [x] Use `deskflow.org` for update check (use GitHub latest release)
- [x] Make app name variable for easier forking
- [x] Move licensing code downstream to Synergy ([discussion #7598](https://github.com/deskflow/deskflow/discussions/7598))
- [x] Release our first version of Deskflow ([v1.17.0](https://github.com/deskflow/deskflow/releases))

### **_Meow'Dib_** (our mascot)
![dune-cat](https://github.com/user-attachments/assets/f9f1dfa9-6622-485a-b9f0-9b91ffff35c4)

**Comment by ccoenen** [comment]
Why was Synergy, a name that has been around for the open source project for more than 20 years, been dropped for the open source part? Why not rename your commercial offering to Deskflow instead?

**Reply by nbolton** [reply]
Good question.

The decision to rename the upstream open source community-driven project to Deskflow stems from a goal to draw a clear line between the open source community efforts and the commercial Synergy product. Over the years, the Synergy brand has evolved into a commercial product in most people's eyes which would be impractical to reverse.  Many now associate the Synergy brand with a commercial licensing model that does not align with a purely community-driven open source project.

Deskflow, on the other hand, is designed to return to open source community roots, focusing on the community's needs without commercial branding or business features. The rebranding reflects an effort to create distinct expectations: Deskflow is purely community-led and free, while Synergy remains the name of the commercial product with additional services and enhancements.

**Reply by torokati44** [reply]
As an outsider, and once-in-maybe-five-years user, my two cents is that: A commercial rebranding can have a huge corporate/legal/etc. overhead, while an open source project renaming itself is mostly a bunch of externalized `sed 's/synergy/deskflow/g'` spread out all across the minds and systems of the volunteers in the OSSverse, followed by "huh, that's a nice new icon I guess 🤷‍♂️".
A very similar thing happened with Redis and Valkey recently, for instance.
Also, I personally like the new name.

**Reply by nbolton** [reply]
Yes good take. From a practical standpoint alone, renaming a GitHub project and replacing a few strings is way more efficient than rebranding a commercial product. Also, Synergy does sound a bit more business-like, I suppose.

**Comment by tamkimd** [comment]
I'm wondering about the security of the Deskflow. Is Deskflow able to connect to the internet?

**Reply by sithlord48** [reply]
It opens a port so if you were connected directly to the internet then yeah. In most cases your going to have to open a port in your routers firewall to enable access outside of your lan.

**Reply by tamkimd** [reply]
> It opens a port so if you were connected directly to the internet then yeah. In most cases your going to have to open a port in your routers firewall to enable access outside of your plan.

Thank you for your answers. 
Is it safe to type or copy important keys?

**Reply by sithlord48** [reply]
If you have it on your lan and your have SSL on you should be good
